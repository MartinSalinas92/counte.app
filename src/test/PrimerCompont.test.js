import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Apiprimer from '../PrimerCompont';

describe('Pruebas en <Apiprimer />', ()=>{

   /* test('debe mostrar el mensaje Hola Mundo', () => {

        const saludo='Hola mundo';

        const {getByText}= render(<Apiprimer saludo={saludo} />)

        expect(getByText(saludo)).toBeInTheDocument();


        
    })*/


    test('Debe de Mostrar <Apiprimer /> correctamente', () => {

        const saludo= 'Hola mundo';
        //shallow es igual al render
        const wrapper= shallow(<Apiprimer saludo={saludo} />);

        expect(wrapper).toMatchSnapshot();
        
    })
    test('Debe Mostrar el subtitulo enviado por el props ', () => {

        const saludo='Hola mundo';
        const subTitulo= 'Soy un subtitulo';

        const wrapper= shallow(
            <Apiprimer 

            saludo={saludo}
            subtitulo={subTitulo}
        />);

        const TextoSubtitulo=wrapper.find('p').text();

        //console.log(TextoSubtitulo);

        expect(TextoSubtitulo).toBe(subTitulo);
        
    })
    
    
    

})