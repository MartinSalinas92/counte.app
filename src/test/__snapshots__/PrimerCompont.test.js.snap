// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Apiprimer /> Debe de Mostrar <Apiprimer /> correctamente 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
